### PR TITLE
chore(acceptance-tests): break up test acceptance-tests-large (--acceptance-go-client-named-vectors)

### LINKED ISSUE #5951: [Tech Debt] Break up test `acceptance-tests-large` (`--acceptance-go-client-named-vectors`)
The test mentioned in the title runs 3 or 4 independent test suites in sequence. This is enough to push it over the default 10m timeout and as a result the timeout has been increased. To make the CICD pipeline faster, it would be better if this test was split up into multiple smaller test jobs that can run in parallel.

### PR DESCRIPTION
**Pull request by melsonic**
### What's being changed:
I have split the `--acceptance-go-client-named-vectors` test into two independent tests to reduce the test running time and make CI/CD pipeline faster
- `--acceptance-go-client-named-vectors-single-node`
- `--acceptance-go-client-named-vectors-cluster`

### Review checklist

- [ ] Documentation has been updated, if necessary. Link to changed documentation:
- [ ] Chaos pipeline run or not necessary. Link to pipeline:
- [ ] All new code is covered by tests where it is reasonable.
- [ ] Performance tests have been run or not necessary.

fixes #5951

### CODE REVIEW

**Comment by antas-marcin** [comment]
@melsonic looks like our `run-swagger` fails on `singlenode` and `cluster` test files. To resolve this please run `./tools/gen-code-from-swagger.sh` script and commit the changes, that should fix the problem.

File: test/acceptance_with_go_client/named_vectors_tests/cluster/cluster_test.go [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+	return
+}
+
+func createClusterEnvironmentAsyncIndexing(ctx context.Context) (compose *docker.DockerCompose, err error) {
+	compose, err = named_vectors_tests_util.ComposeModules().
+		WithWeaviateEnv("ASYNC_INDEXING", "true").
+		WithWeaviateCluster().
+		Start(ctx)
+	return
+}
```
Comment (antas-marcin) [inline test/acceptance_with_go_client/named_vectors_tests/cluster/cluster_test.go:60]: please add an empty line at the end of the file

**Comment by melsonic** [comment]
> @melsonic looks like our `run-swagger` fails on `singlenode` and `cluster` test files. To resolve this please run `./tools/gen-code-from-swagger.sh` script and commit the changes, that should fix the problem.

Sure, I'll run that script before the next commit.

**Comment by melsonic** [comment]
> @melsonic and can we alter the structure a little bit and put the tests into their package, then the structure would look like this:
> 
> ```
> - named_vectors_tests
>     |- cluster
>     |- singlenode
>     |- tests (or test_suits)
> ```

Yeah, we can definitely do that, that will also make the file structure a bit cleaner.

**Comment by melsonic** [comment]
Hey @antas-marcin, I have changed the folder structure as per your suggestion and ran the `./tools/gen-code-from-swagger.sh` script before committing the changes. Let me know if you have any further suggestions to improve the quality of the PR.

And one more question just out of curiosity, does weaviate follow adding a newline at the end of the file because of this reason (`Many older tools misbehave if the last line of data in a text file is not terminated with a newline or carriage return / new line combination. They ignore that line as it is terminated with ^Z (eof) instead.`)? 
I found this on a StackOverflow thread.

Thank you

**Comment by antas-marcin** [comment]
@melsonic looks great now! thank you for your contribution! I will merge it once the CI will be green. Once again thank you!

**Comment by melsonic** [comment]
> @melsonic looks great now! thank you for your contribution! I will merge it once the CI will be green. Once again thank you!

My pleasure @antas-marcin

**Review by antas-marcin — APPROVED** [review]

**Comment by antas-marcin** [comment]
@melsonic one last thing I don't know if you already agreed to our CLA or not, if not please just comment add a comment here that you do :) simply adding a comment with "I agree to CLA" will do the trick. Thank you!

**Comment by melsonic** [comment]
@antas-marcin most probably, I did. But to avoid any confusion I am adding the comment here again.

**Comment by melsonic** [comment]
I agree to CLA

**Comment by antas-marcin** [comment]
Thank you!

**Pull request merged**